apps/io-session-manager-internal/src/controllers/__tests__/info.test.ts

```diff
@@ -1,30 +1,53 @@
-import { describe, expect, it, vi } from "vitest";
+import { describe, expect, it, Mock } from "vitest";
 import * as E from "fp-ts/lib/Either";
 import * as H from "@pagopa/handler-kit";
 import { makeInfoHandler } from "../info";
-import * as packageUtils from "../../utils/package";
 import { httpHandlerInputMocks } from "../__mocks__/handler.mock";
-
-const getCurrentBackendVersionMock = vi.spyOn(
-  packageUtils,
-  "getCurrentBackendVersion",
-);
+import { mockInfoService } from "../__mocks__/info-service.mock";
+import { aPackageInfo, mockPackageUtils } from "../../__mocks__/package.mock";
 
 describe("Info handler", () => {
   it("should succeed if the application is healthy", async () => {
-    getCurrentBackendVersionMock.mockReturnValueOnce("1.0.0");
+    const result = await makeInfoHandler({
+      ...httpHandlerInputMocks,
+      InfoService: mockInfoService,
+      PackageUtils: mockPackageUtils,
```

Comment:
Controllers shouldn't need this, because InfoService is already mocked. replace also other references in this file
```suggestion
      PackageUtils: {} as PackageUtils,
```

Reply:
Resolved in f2cae0e78fe0b03e6e7c59b5b01cdbde960c79c8